db/src/main/resources/db/migration/V1_018__add_payment_tables.sql

```diff
@@ -0,0 +1,35 @@
+create table payment_cycle(
+    id uuid not null primary key,
+    version_number int not null,
+    claim_id uuid not null references claim(id),
+    card_account_id varchar(50) not null,
+    cycle_start_date timestamp,
+    cycle_end_date timestamp,
+    eligibility_status varchar(50),
+    voucher_entitlement_json json,
+    expected_delivery_date timestamp,
+    children_dob_json json,
+    total_vouchers int,
+    total_entitlement_amount_in_pence int,
+    card_balance_in_pence int default 0,
```

Comment:
Do you also want to default the ints above to zero, or is there a reason why you've specifically defaulted only this one?

Reply:
Removed.